### PR TITLE
AAF Experiments

### PR DESCRIPTION
**Pull request by andrewhall1124**
- Updated training configs for 7 different models.
- New testing configs for all 7 models.

### CODE REVIEW

File: probcal/models/faithful_gaussian_nn.py [hunk 1]
```diff
@@ -28,8 +28,8 @@ def __init__(
         self,
         backbone_type: Type[Backbone],
         backbone_kwargs: dict,
-        optim_type: OptimizerType,
-        optim_kwargs: dict,
+        optim_type: OptimizerType = None,
+        optim_kwargs: dict = None,
```
Comment (spencermyoung513) [inline probcal/models/faithful_gaussian_nn.py]: Can we update the types on these so they're typed optionally? It'll look like

```
optim_type: OptimizerType | None = None,
optim_kwargs: dict | None = None,
```

Comment (andrewhall1124) [reply probcal/models/faithful_gaussian_nn.py]: Just added this for faithful gaussian, natural gaussian, and negative binomial

File: configs/test/aaf/gaussian_beta_cfg.yaml
Comment (spencermyoung513) [inline configs/test/aaf/gaussian_beta_cfg.yaml:1]: Can we change all references to `gaussian_beta` to `seitzer`? We've been using that in other places/checkpoints to refer to this model type

**Review by spencermyoung513 — APPROVED** [review]

File: configs/test/aaf/gaussian_beta_cfg.yaml
Comment (andrewhall1124) [reply configs/test/aaf/gaussian_beta_cfg.yaml:1]: These changes have been completed.

**Pull request merged**